addons/l10n_it/models/account_report.py

```diff
@@ -10,6 +10,6 @@ class AccountReportExpression(models.AbstractModel):
     def _get_carryover_target_expression(self, options):
         if self.report_line_id.code == 'VP14b' and fields.Date.from_string(options['date']['date_to']).month == 12:
             # For this line, if we are between two years, we want to carry over to vp9 instead of the line set in the XML file (vp8)
-            return self.env.ref('l10n_it.tax_report_line_vp9_applied_carryover')
+            return self.env.ref('l10n_it.tax_monthly_report_line_vp9_applied_carryover')
```

Comment:
This needs to work for both reports.
The ID needs to be different depending on `self`